official/utils/logging/examples_per_second_hook_test.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os.path
+import shutil
+import tempfile
+import time
+
+from tensorflow.python.training import monitored_session
```

Comment:
I think you should be able to get to this from the public API as well (tf....); is that not true?

Reply:
Not really. I checked the monitored_session module, and the required _HookSessions() is a protected class without any wrapper for high-level API for now. Will double check it and look for other possible alternative APIs.